src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs

```diff
@@ -9,15 +9,42 @@ namespace Microsoft.ML.Runtime.Internal.CpuMath
 {
     public static partial class CpuMathUtils
     {
+        private static string _enableAvx = Environment.GetEnvironmentVariable("COMPlus_EnableAVX");
```

Comment:
You don't need to add these environment variable checks.  The .NET runtime does this checking.

You should just have `if (Avx.IsSupported) {} else if (Sse.IsSupported) {}` directly in your methods below, like you had before.